packages/xo-web/src/xo-app/hub/resource.js

```diff
@@ -171,10 +180,66 @@ export default decorate([
       redirectToTaskPage() {
         this.props.router.push('/tasks')
       },
+      showDescription() {
+        const {
+          data: { public: _public },
+          name,
+        } = this.props
+        alert(
+          name,
+          <div>
+            {isEmpty(omit(_public, BANNED_FIELDS)) ? (
+              <p>{_('hubTemplateDescriptionNotAvailable')}</p>
+            ) : (
+              <div>
+                <Ul>
+                  {EXCLUSIVE_FIELDS.map(fieldKey => {
+                    const field = _public[fieldKey]
+                    return field !== undefined ? (
+                      <Li key={fieldKey}>{field}</Li>
+                    ) : null
+                  })}
+                </Ul>
+                <br />
+                <Ul>
+                  {map(omit(_public, STATIC_FIELDS), (value, key) => (
+                    <Li key={key}>
+                      {startCase(key).toLowerCase()}
```

Comment:
`_.startCase` is supposed to add capital letters at the beginning of each word but I'm seeing all lower case in the screenshot.

Reply:
because I added `.toLowerCase()` in the end... do you want'em starting with capital letters ?

Reply:
Oh right! No, it's fine like this.